### DISCUSSION TITLE
Ignore baseBranches for certain dependencies

### DISCUSSION THREAD

**Discussion by Sebazzz** [renovatebot/renovate#27902] [Request Help]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Gitea

### Please tell us more about your question or problem

In general, I want any dependency update to be made against my default branch. That goes well in the default Renovate config.

However, for certain updates (like dotnet/efcore) I might want to ignore that `baseBranches` and explicitly also allow release branches to be updated. 

Is there any way this can be done in renovate config?

**Comment by rarkins** [comment]
Yes, it can be done. You need to define baseBranches and then packageRules with matchBaseBranches to enable/disable packages in each as you need
